### PR TITLE
Improve UX

### PR DESCRIPTION
**Pull request by lukasschreiber**
Closes https://github.com/reform-org/reform/issues/269
Closes https://github.com/reform-org/reform/issues/171
Closes https://github.com/reform-org/reform/issues/314
Closes https://github.com/reform-org/reform/issues/275
Closes https://github.com/reform-org/reform/issues/264
Closes https://github.com/reform-org/reform/issues/315
Closes https://github.com/reform-org/reform/issues/281
Closes https://github.com/reform-org/reform/issues/277

### CODE REVIEW

File: js/src/main/scala/webapp/services/RoutingService.scala [hunk 1]
```diff
@@ -52,9 +52,14 @@ class RoutingService(using repositories: Repositories, toaster: Toaster) {
   ): Signal[VNode] =
     page.map(_.render)
 
-  def to(newPage: Page, preventReturn: Boolean = false) = {
-    window.history.pushState(null, "", linkPath(newPage))
-    page.set(newPage)
+  def to(newPage: Page, preventReturn: Boolean = false, newTab: Boolean = false) = {
```
Comment (mohe2015) [inline js/src/main/scala/webapp/services/RoutingService.scala:55]: I think we can remove the preventReturn attribute as it should be unused

File: js/src/main/scala/webapp/components/NavigationLink.scala [hunk 2]
```diff
@@ -30,7 +30,11 @@ def navigationLink(using routing: RoutingService)(page: Page, label: String): VN
     onClick.foreach(e => {
       e.preventDefault()
       e.target.asInstanceOf[HTMLElement].blur()
-      routing.to(page, true)
+      if (e.ctrlKey) {
+        routing.to(page, true, true)
```
Comment (mohe2015) [inline js/src/main/scala/webapp/components/NavigationLink.scala:34]: would an alternative fix be to not call preventDefault and blur if you pressed ctrlKey?

Comment (mohe2015) [reply js/src/main/scala/webapp/components/NavigationLink.scala:34]: I think that would be better as it would use more of the browsers default behaviour

**Comment by lukasschreiber** [comment]
Actions column should be way smaller on mobile

Dropdowns in table are getting cut off overflow y visible should help

**Comment by lukasschreiber** [comment]
Bugs that need to be addressed before merging:

- [ ] Scrolling the table horizontally does not work
- [ ] popper places the dropdown at the wrong offset (ca. 8px too high)
- [ ] Dropdowns do not close on click outside, UI (especially mobile menu) is therefore not navigatable because other dropdowns overlay it.
- [ ] td does not has max width but div inside, therefore some tds are too large on big screens

**Comment by lukasschreiber** [comment]
- [ ] Multiselect closes once option has been selected
- [ ] multiclose in table: popper misplaced until scroll

**Pull request merged**